pkg/validator/cluster.go

```diff
@@ -832,6 +833,64 @@ func validateNamingScheme(
 	return patches
 }
 
+// normalizeResourceQuantities is called by normalizeRolesResourceQuantities
+// to handle each role resource list. If the quantity of a resource is
+// equivalent between old and new lists, then copy the new value into the old
+// list. If any resources have been removed or added, this function will
+// bail out early returning false; otherwise it returns true once done.
+func normalizeResourceQuantities(
+	prevList corev1.ResourceList,
```

Comment:
don't know why they call this ResourceList it should be ResourceMap :)

Reply:
yah